### ISSUE TITLE
No notification

### ISSUE THREAD

**Issue by 03397** [hacs/ND-NotifyUpdates#1]
I have installed this but I never get any notification when something is updated in HACS

**Comment by ludeeus** [comment]
What does the netdaemon logs say?

**Comment by 03397** [comment]
Where can I find this log?
Is the netdaemon a separate installation from HACS and from Home Assistant?
I just installed HACS and then I installed the HACS Update Notifications from the repo.

**Comment by ludeeus** [comment]
You need to be running this: https://netdaemon.xyz/ to use NetDaemon apps.
If you don't want that, there are HA automation examples here: https://hacs.xyz/docs/basic/automation

**Comment by nathang21** [comment]
I've never had this work, but I do have the NetDaemon runtime addon as well as the NetDaemon integration on HACS
 running.
 
 Logs:
 ```
 [s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
Starting NetDaemon Admin
Starting NetDaemon Runner
[services.d] done.
Running standard NetDaemon at /daemon...
Admin GUI: NetDaemon API not available yet, error:  connect ECONNREFUSED 127.0.0.1:5000 Closing and retrying...
Admin GUI: Reconnect will be attempted in 10 seconds. 
[08:02:09 ERR] NetDaemon.Service.App.DaemonAppCompiler: Compiler error!
/config/netdaemon/apps/HacsNotifyOnUpdate/HacsNotifyOnUpdate.cs(7,35): error CS0246: The type or namespace name 'NetDaemonApp' could not be found (are you missing a using directive or an assembly reference?)
/config/netdaemon/apps/HacsNotifyOnUpdate/HacsNotifyOnUpdate.cs(3,7): error CS0246: The type or namespace name 'JoySoftware' could not be found (are you missing a using directive or an assembly reference?)
/config/netdaemon/apps/HacsNotifyOnUpdate/HacsNotifyOnUpdate.cs(9,36): error CS0115: 'NotifyOnUpdate.InitializeAsync()': no suitable method found to override
/config/netdaemon/apps/HacsNotifyOnUpdate/HacsNotifyOnUpdate.cs(9,57): error CS0103: The name 'Entity' does not exist in the current context

[08:02:09 INF] NetDaemon.Service.RunnerService: Starting NeDaemon service (version 21.51.0)...
[08:02:10 INF] NetDaemon.Daemon.NetDaemonHost: Successfully connected to Home Assistant Core in Home Assistant Add-on
[08:02:11 ERR] NetDaemon.Service.App.DaemonAppCompiler: Compiler error!
/config/netdaemon/apps/HacsNotifyOnUpdate/HacsNotifyOnUpdate.cs(7,35): error CS0246: The type or namespace name 'NetDaemonApp' could not be found (are you missing a using directive or an assembly reference?)
/config/netdaemon/apps/HacsNotifyOnUpdate/HacsNotifyOnUpdate.cs(3,33): error CS0234: The type or namespace name 'NetDaemon' does not exist in the namespace 'JoySoftware.HomeAssistant' (are you missing an assembly reference?)
/config/netdaemon/apps/HacsNotifyOnUpdate/HacsNotifyOnUpdate.cs(9,36): error CS0115: 'NotifyOnUpdate.InitializeAsync()': no suitable method found to override
/config/netdaemon/apps/HacsNotifyOnUpdate/HacsNotifyOnUpdate.cs(9,57): error CS0103: The name 'Entity' does not exist in the current context

[08:02:11 ERR] NetDaemon.Service.RunnerService: No NetDaemon apps could be found, exiting...
[08:02:11 INF] NetDaemon.Service.RunnerService: NetDaemon service exited!
Admin GUI: Running NetDaemon Admin
Admin GUI: received message from frontend: apps
Admin GUI: received message from frontend: settings
Admin GUI: received message from NetDaemon: apps
Admin GUI: received message from NetDaemon: settings
Admin GUI: received message from frontend: apps
Admin GUI: received message from frontend: settings
...
```
